mne/viz/tests/test_evoked.py

```diff
@@ -334,7 +341,7 @@ def test_plot_compare_evokeds():
     red.crop(0.01, 0.01)
     assert len(red.times) == 1
     plot_compare_evokeds(red)
-    plt.close('all')
+    plt.close('all')    
```

Comment:
boooo hissss PEP8

Reply:
Added that in the `fix pep` commit too

Reply:
haha, so ironic :D BTW thanks for pinging me, Jona, I'll take a look today.